src/components/App/App.js

```diff
@@ -1,10 +1,90 @@
 import styled from 'styled-components/macro'
 import Countdown from '../Countdown/Countdown'
+import { ReactComponent as PlayButton } from '../../assets/play-icon.svg'
+import { ReactComponent as StopButton } from '../../assets/stop-icon.svg'
+import { useState, useEffect } from 'react'
 
 function App() {
+  const LENGTHTWENTYFIVE = { minutes: 25, seconds: 0 }
+  const LENGTHFIFTY = { minutes: 50, seconds: 0 }
+
+  const [lengthFifty, setLengthFifty] = useState(false)
+  const [isActive, setIsActive] = useState(false)
+  const [timerExpired, setTimerExpired] = useState(false)
+  const [[minutes, seconds], setCounter] = useState([
+    LENGTHTWENTYFIVE.minutes,
+    LENGTHTWENTYFIVE.seconds,
+  ])
+
+  function timer() {
+    if (timerExpired) return
+    if (minutes === 0 && seconds === 0) setTimerExpired(true)
+    else if (seconds === 0) {
+      setCounter([minutes - 1, 59])
+    } else {
+      setCounter([minutes, seconds - 1])
+    }
+  }
+
+  useEffect(() => {
+    if (isActive) {
+      const timeoutID = setTimeout(() => timer(), 1000)
```

Comment:
Try `requestAnimationFrame(timestamp)` with current time to be able to track time passed when mobile screen is not active.
See more at [MDN](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Timeouts_and_intervals)